### ISSUE TITLE
Investigate why the model loss curve diverges between train-val and test datasets.

### ISSUE THREAD

**Issue by eulersson** [eulersson/taconez#111]
By looking at the model loss across the epochs it seems we are overfitting.

![Image](https://github.com/eulersson/taconez/assets/6486812/8d048437-5a62-4e61-8037-e2f19fc19414)

Investigate how to fix that.

**Comment by eulersson** [comment]
Sometimes the loss curve is fine though! Look:

![Image](https://github.com/eulersson/taconez/assets/6486812/a5405af5-89e4-46d1-8100-3b2ac89144ff)

So I guess it might be because there are too little samples in the test dataset.
